BioArchLinux/ibs/IBS.desktop

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env xdg-open
```

Comment:
Shebang is not needed in desktop files according to my knowledge. If I am wrong , please show me a reference